Explicitly include cstdint as required by gcc13.  This is a temporary measure while
upstream makes a long-term fix.

Upstream-Status: Pending

Index: git/src/address_range.cpp
===================================================================
--- git.orig/src/address_range.cpp
+++ git/src/address_range.cpp
@@ -27,6 +27,7 @@
  *
  */
 
+#include <cstdint>
 #include <tins/address_range.h>
 #include <tins/ip_address.h>
 #include <tins/ipv6_address.h>
Index: git/src/arp.cpp
===================================================================
--- git.orig/src/arp.cpp
+++ git/src/arp.cpp
@@ -27,6 +27,7 @@
  *
  */
 
+#include <cstdint>
 #include <cstring>
 #include <tins/arp.h>
 #include <tins/ethernetII.h>
Index: git/src/bootp.cpp
===================================================================
--- git.orig/src/bootp.cpp
+++ git/src/bootp.cpp
@@ -27,6 +27,7 @@
  *
  */
 
+#include <cstdint>
 #include <cstring>
 #include <tins/bootp.h>
 #include <tins/exceptions.h>
Index: git/src/detail/address_helpers.cpp
===================================================================
--- git.orig/src/detail/address_helpers.cpp
+++ git/src/detail/address_helpers.cpp
@@ -27,6 +27,7 @@
  *
  */
 
+#include <cstdint>
 #include <tins/ip_address.h>
 #include <tins/ipv6_address.h>
 #include <tins/endianness.h>
Index: git/src/detail/pdu_helpers.cpp
===================================================================
--- git.orig/src/detail/pdu_helpers.cpp
+++ git/src/detail/pdu_helpers.cpp
@@ -27,6 +27,7 @@
  *
  */
 
+#include <cstdint>
 #include <tins/detail/pdu_helpers.h>
 #ifdef TINS_HAVE_PCAP
     #include <pcap.h>
Index: git/src/dhcp.cpp
===================================================================
--- git.orig/src/dhcp.cpp
+++ git/src/dhcp.cpp
@@ -27,6 +27,7 @@
  *
  */
 
+#include <cstdint>
 #include <cstring>
 #include <tins/endianness.h>
 #include <tins/dhcp.h>
Index: git/src/dns.cpp
===================================================================
--- git.orig/src/dns.cpp
+++ git/src/dns.cpp
@@ -27,6 +27,7 @@
  *
  */
 
+#include <cstdint>
 #include <utility>
 #include <cstdio>
 #include <tins/dns.h>
Index: git/src/dot11/dot11_base.cpp
===================================================================
--- git.orig/src/dot11/dot11_base.cpp
+++ git/src/dot11/dot11_base.cpp
@@ -27,6 +27,7 @@
  *
  */
 
+#include <cstdint>
 #include <tins/dot11/dot11_base.h>
 
 #ifdef TINS_HAVE_DOT11
Index: git/src/dot3.cpp
===================================================================
--- git.orig/src/dot3.cpp
+++ git/src/dot3.cpp
@@ -27,6 +27,7 @@
  *
  */
 
+#include <cstdint>
 #include <cstring>
 #include <tins/macros.h>
 #ifndef _WIN32
Index: git/src/ethernetII.cpp
===================================================================
--- git.orig/src/ethernetII.cpp
+++ git/src/ethernetII.cpp
@@ -27,6 +27,7 @@
  *
  */
 
+#include <cstdint>
 #include <cstring>
 #include <tins/macros.h>
 #ifndef _WIN32
Index: git/src/icmp.cpp
===================================================================
--- git.orig/src/icmp.cpp
+++ git/src/icmp.cpp
@@ -27,6 +27,7 @@
  *
  */
 
+#include <cstdint>
 #include <cstring>
 #ifndef _WIN32
     #include <netinet/in.h>
Index: git/src/ip_address.cpp
===================================================================
--- git.orig/src/ip_address.cpp
+++ git/src/ip_address.cpp
@@ -27,6 +27,7 @@
  *
  */
 
+#include <cstdint>
 #ifdef _WIN32
     #include <ws2tcpip.h>
 #else // _WIN32
Index: git/src/ip.cpp
===================================================================
--- git.orig/src/ip.cpp
+++ git/src/ip.cpp
@@ -27,6 +27,7 @@
  *
  */
 
+#include <cstdint>
 #include <cstring>
 #ifndef _WIN32
     #include <netdb.h>
Index: git/src/ip_reassembler.cpp
===================================================================
--- git.orig/src/ip_reassembler.cpp
+++ git/src/ip_reassembler.cpp
@@ -27,6 +27,7 @@
  *
  */
 
+#include <cstdint>
 #include <tins/ip.h>
 #include <tins/constants.h>
 #include <tins/ip_reassembler.h>
Index: git/src/ipv6.cpp
===================================================================
--- git.orig/src/ipv6.cpp
+++ git/src/ipv6.cpp
@@ -27,6 +27,7 @@
  *
  */
 
+#include <cstdint>
 #include <cstring>
 #ifndef _WIN32
     #include <netinet/in.h>
Index: git/src/loopback.cpp
===================================================================
--- git.orig/src/loopback.cpp
+++ git/src/loopback.cpp
@@ -27,6 +27,7 @@
  *
  */
 
+#include <cstdint>
 #ifndef _WIN32
     #include <sys/socket.h>
     #ifdef BSD
Index: git/src/memory_helpers.cpp
===================================================================
--- git.orig/src/memory_helpers.cpp
+++ git/src/memory_helpers.cpp
@@ -27,6 +27,7 @@
  *
  */
 
+#include <cstdint>
 #include <tins/memory_helpers.h>
 #include <tins/ip_address.h>
 #include <tins/ipv6_address.h>
Index: git/src/mpls.cpp
===================================================================
--- git.orig/src/mpls.cpp
+++ git/src/mpls.cpp
@@ -27,6 +27,7 @@
  *
  */
 
+#include <cstdint>
 #include <tins/mpls.h>
 #include <tins/ip.h>
 #include <tins/ipv6.h>
Index: git/src/network_interface.cpp
===================================================================
--- git.orig/src/network_interface.cpp
+++ git/src/network_interface.cpp
@@ -27,6 +27,7 @@
  *
  */
 
+#include <cstdint>
 #include <stdexcept>
 #include <vector>
 #include <cstring>
Index: git/src/packet_sender.cpp
===================================================================
--- git.orig/src/packet_sender.cpp
+++ git/src/packet_sender.cpp
@@ -27,6 +27,7 @@
  *
  */
 
+#include <cstdint>
 #include <tins/packet_sender.h>
 #ifndef _WIN32
     #include <sys/socket.h>
Index: git/src/pdu.cpp
===================================================================
--- git.orig/src/pdu.cpp
+++ git/src/pdu.cpp
@@ -27,6 +27,7 @@
  *
  */
  
+#include <cstdint>
 #include <tins/pdu.h>
 #include <tins/packet_sender.h>
 
Index: git/src/pdu_option.cpp
===================================================================
--- git.orig/src/pdu_option.cpp
+++ git/src/pdu_option.cpp
@@ -27,6 +27,7 @@
  *
  */
 
+#include <cstdint>
 #include <algorithm>
 #include <tins/pdu.h>
 #include <tins/ip_address.h>
Index: git/src/radiotap.cpp
===================================================================
--- git.orig/src/radiotap.cpp
+++ git/src/radiotap.cpp
@@ -27,6 +27,7 @@
  *
  */
 
+#include <cstdint>
 #include <tins/radiotap.h>
 
 #ifdef TINS_HAVE_DOT11
Index: git/src/tcp.cpp
===================================================================
--- git.orig/src/tcp.cpp
+++ git/src/tcp.cpp
@@ -27,6 +27,7 @@
  *
  */
 
+#include <cstdint>
 #include <cstring>
 #include <tins/tcp.h>
 #include <tins/ip.h>
Index: git/src/tcp_ip/flow.cpp
===================================================================
--- git.orig/src/tcp_ip/flow.cpp
+++ git/src/tcp_ip/flow.cpp
@@ -27,6 +27,7 @@
  *
  */
 
+#include <cstdint>
 #include <tins/tcp_ip/flow.h>
 
 #ifdef TINS_HAVE_TCPIP
Index: git/src/tcp_ip/stream_identifier.cpp
===================================================================
--- git.orig/src/tcp_ip/stream_identifier.cpp
+++ git/src/tcp_ip/stream_identifier.cpp
@@ -27,6 +27,7 @@
  *
  */
 
+#include <cstdint>
 #include <tins/tcp_ip/stream_identifier.h>
 
 #ifdef TINS_HAVE_TCPIP
Index: git/src/tcp_stream.cpp
===================================================================
--- git.orig/src/tcp_stream.cpp
+++ git/src/tcp_stream.cpp
@@ -27,6 +27,7 @@
  *
  */
 
+#include <cstdint>
 #include <limits>
 #include <tins/rawpdu.h>
 #include <tins/tcp_stream.h>
Index: git/src/udp.cpp
===================================================================
--- git.orig/src/udp.cpp
+++ git/src/udp.cpp
@@ -27,6 +27,7 @@
  *
  */
 
+#include <cstdint>
 #include <cstring>
 #include <tins/udp.h>
 #include <tins/constants.h>
Index: git/src/utils/checksum_utils.cpp
===================================================================
--- git.orig/src/utils/checksum_utils.cpp
+++ git/src/utils/checksum_utils.cpp
@@ -27,6 +27,7 @@
  *
  */
 
+#include <cstdint>
 #include <tins/utils/checksum_utils.h>
 #include <cstring>
 #include <tins/ip_address.h>
Index: git/src/utils/resolve_utils.cpp
===================================================================
--- git.orig/src/utils/resolve_utils.cpp
+++ git/src/utils/resolve_utils.cpp
@@ -27,6 +27,7 @@
  *
  */
 
+#include <cstdint>
 #include <tins/utils/resolve_utils.h>
  #ifndef _WIN32
     #if defined(BSD) || defined(__FreeBSD_kernel__)
Index: git/src/utils/routing_utils.cpp
===================================================================
--- git.orig/src/utils/routing_utils.cpp
+++ git/src/utils/routing_utils.cpp
@@ -27,6 +27,7 @@
  *
  */
 
+#include <cstdint>
 #include <tins/utils/routing_utils.h>
 #ifndef _WIN32
     #if defined(BSD) || defined(__FreeBSD_kernel__)
